toplevel/vernac.ml

```diff
@@ -145,6 +152,7 @@ let rec interp_vernac sid po (loc,com) =
       let com = if !Flags.time then VernacTime (loc,com) else com in
       interp com
     with reraise ->
+      ignore(Stm.edit_at sid);
```

Comment:
If `interp` fails, we go back to the previous state. This seems clear to me right?